RAMP/common.py

```diff
@@ -1,7 +1,14 @@
 
 from __future__ import print_function
 import sys
+import os
 
+try:
+    sys.path.insert(0, os.path.join(os.path.dirname(__file__), "../../readies"))
+    sys.path.insert(0, "/w/rafi_1/readies")
```

Comment:
Why you need this? Why readies is needed in ramp?

Reply:
It helps with debugging (lets you set breakpoints via `BB()`). Anyways, the line with the `/w/rafi_1/readies` shouldn't be there at all.
I can remove it if it's a pain in the eye.

Reply:
Maybe we should remove all of them if they are not needed no?

Reply:
I'll comment it out, so it'll be easy to restore when needed. Anyway, there's no much sense in looking for `readies` in the filesystem when RAMP is installed with pip.